app/views/content/home/_calls-to-action.html.erb

```diff
@@ -1,11 +1,11 @@
 <div class="homepage-feature blocks">
   <%= render CallsToAction::HomepageComponent.new(
     icon: "icon-calendar",
-    title: "Attend an event",
-    text: "Talk to course providers, teachers and expert advisers to get your questions answered in person or online at an event.",
-    link_text: "Find an event",
-    link_target: "/events",
-    image: "media/images/content/homepage/here-to-help.jpg")
+    title: "Get personalised guidance to your inbox",
+    text: "Answer a few questions about your current situation and we'll send you helpful advice and insights relevant to you.",
+    link_text: "Register for updates",
```

Comment:
Is updates the right term here @EllieNodder ? Might be worth trying out something else as we know that people were confused by this in UR - and it doesn't seem to fit with the content before it